clang/test/Analysis/stack-addr-ps.c

```diff
@@ -126,3 +126,22 @@ void caller_for_nested_leaking() {
   int *ptr = 0;
   caller_mid_for_nested_leaking(&ptr);
 }
+
+// This used to crash StackAddrEscapeChecker because
+// it features a symbol conj_$1{struct c *, LC1, S763, #1}
+// that has no origin region.
+// bbi-98571
```

Comment:
Tests are not supposed to have references to tickets that are not publicly reachable. Swap it for a link if you really want to preserve this.

Reply:
removed in 7cd8719